lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1100,37 +1086,24 @@
     }
 
     .bordered-bar {
-
       h2 {
         font-size: 14px;
       }
 
       span {
-        float: right;
+        @include float(right);
       }
     }
-    .user-data {
-
-      .data-input {
 
-        h1 {
-          font-size: 18px;
-        }
-      }
+    .user-data .data-input h1 {
+      font-size: 18px;
     }
 
     table.course-receipt {
-
-      tr {
-
-        td {
-
-          a {
-
-            &:before {
-              content:" " attr(data-base-url) " ";
-              display: inline-block;}
-          }
+      tr td a {
```

Comment:
+1. cleaner nesting here. Class name here would be even better, but this is helpful cleanup as is.